.env-devel

```diff
@@ -30,6 +30,7 @@ AUTOSCALING_LOGLEVEL=WARNING
 AUTOSCALING_NODES_MONITORING=null
 AUTOSCALING_POLL_INTERVAL=10
 
+CATALOG_BACKGROUND_TASK_REST_TIME=60
```

Comment:
I cannot find this env var in `docker-compose.yml`. Is it on purpose?

Reply:
It was already there. The new testing detected that it was not in `.env-devel`
![image](https://github.com/ITISFoundation/osparc-simcore/assets/32402063/f5bed682-3644-4d4e-9573-d521617571b9)